### DISCUSSION TITLE
Is there a rule to detect unknown arguments in docstring?

### DISCUSSION THREAD

**Discussion by harupy** [astral-sh/ruff#13408] [Q&A] [answered]
In the following example, `f` only has `x` but the docstring contains `y`. Is there a rule to detect unknown arguments like `y`?

```python
def f(x: str):
    """
    Args:
        x: string
        y: another string  <- unknown argument

    """
    ...
```

**Comment by harupy** [comment]
Looks like there isn't:

https://play.ruff.rs/8033a40e-859c-41f8-be19-2f2bde422658

**Reply by MichaReiser** [reply, marked as answer]
I think you also have to set `preview` and a `convention`, but there's no such rule. There's https://github.com/astral-sh/ruff/pull/13280, but we must figure out what to do with the two overlapping rules. We're working on it, but it may take a bit longer.

**Reply by harupy** [reply]
thanks for the quick reply!
